internal/setup-smoke-test/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/DataDog/dd-trace-go/internal/setup-smoke-test
+
+go 1.22
```

Comment:
```suggestion
go 1.22.0
```

Reply:
See https://go.dev/play/p/KkX12rqpSwI - I'll update this PR.

Reply:
83b630a5e